distribution/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>parent</artifactId>
         <groupId>org.igniterealtime.openfire</groupId>
-        <version>4.6.0</version>
+        <version>4.6.1-SNAPSHOT</version>
```

Comment:
Changing version identifiers as used in the Maven infrastructure might be desirable, but has nothing to do with the changes suggested in this PR. Please take them out. By doing that, the PR will be a lot easier to review and merge, which helps a lot in us reviewing and merging it.

As an aside: these changes are already in the code: you'll find that similar changes have already been made on the current HEAD of the primary branch.

Reply:
@guusdk  hi first of all am sorry replying late. I found you are correct and it happened because when i check out before commit from 4.5 to 4.6 my codes conflicted. so i checked it again and now it was just one more condition need to added in first IF case and thats we want to run when somebody modify from openfire admin portal.